### ISSUE TITLE
Callback

### ISSUE THREAD

**Issue by fyodorvi** [jackfranklin/jspm-dev-builder#17]
Thanks for the module, it works like charm.

Is there a way to provide a callback for build method for better gulp integration?

**Comment by fyodorvi** [comment]
Looked into the code, build() actually returns a promise. Please, add it to the doc.

**Comment by jackfranklin** [comment]
Yup, build does indeed return a promise. I'd love a PR to update the docs if you can :)
